llvm/include/llvm/Object/ELFObjectFile.h

```diff
@@ -110,11 +110,12 @@ class ELFObjectFileBase : public ObjectFile {
   Expected<std::vector<VersionEntry>> readDynsymVersions() const;
 
   /// Returns a vector of all BB address maps in the object file. When
-  // `TextSectionIndex` is specified, only returns the BB address maps
-  // corresponding to the section with that index. When `PGOAnalyses`is
-  // specified, the vector is cleared then filled with extra PGO data.
-  // `PGOAnalyses` will always be the same length as the return value on
-  // success, otherwise it is empty.
+  /// `TextSectionIndex` is specified, only returns the BB address maps
+  /// corresponding to the section with that index. When `PGOAnalyses`is
+  /// specified, the vector is cleared then filled with extra PGO data.
+  /// When PGO information is requested (`PGOAnalyses` is not nullptr),
+  /// `PGOAnalyses` will always be the same length as the return value,
+  /// assuming no error occurs. Upon failure, `PGOAnalyses` will be emptied.
```

Comment:
Could we merge the ``When `PGOAnalyses` is specified, ...`` and ``When PGO information is requested (`PGOAnalyses` is not nullptr),``. Seeing two very similar phrases start of two consecutive sentences threw me off. Maybe just have it on the first sentence or combine the sentences.

Reply:
Good point. Reworded it to only keep that phrasing in the first sentence.